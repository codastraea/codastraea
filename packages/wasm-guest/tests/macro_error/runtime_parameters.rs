use serpent_automation_wasm_guest::workflow;

#[workflow]
async fn counter(x: usize) {}

fn main() {}
