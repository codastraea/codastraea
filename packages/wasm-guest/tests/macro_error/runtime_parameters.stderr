error: `workflow` functions should have no runtime parameters
 --> tests/macro_error/runtime_parameters.rs:4:18
  |
4 | async fn counter(x: usize) {}
  |                  ^^^^^^^^
